index.html

```diff
@@ -204,55 +229,2313 @@ <h3>
+        </li>
+        <li>Initialize |usages| to be an empty [=sequence=] of `unsigned long`.
+        </li>
+        <li>Initialize |stringIndices| to be an empty [=list=].
+        </li>
+        <li>Initialize |globalStack| to be an empty [=stack=].
+        </li>
+        <li>Initialize |initialGlobalState| to be an empty [=ordered map=].
+        </li>
+        <li>Initialize |reportId| to 0.
+        </li>
+        <li>Set |initialGlobalState|["{{HIDReportItem/unitSystem}}"] to
+        {{HIDUnitSystem/"none"}}.
+        </li>
+        <li>Set |initialGlobalState|["{{HIDReportItem/unitExponent}}"] to 0.
+        </li>
+        <li>Set
+        |initialGlobalState|["{{HIDReportItem/unitFactorLengthExponent}}"] to
+        0.
+        </li>
+        <li>Set
+        |initialGlobalState|["{{HIDReportItem/unitFactorMassExponent}}"] to 0.
+        </li>
+        <li>Set
+        |initialGlobalState|["{{HIDReportItem/unitFactorTimeExponent}}"] to 0.
+        </li>
+        <li>Set
+        |initialGlobalState|["{{HIDReportItem/unitFactorTemperatureExponent}}"]
+        to 0.
+        </li>
+        <li>Set
+        |initialGlobalState|["{{HIDReportItem/unitFactorCurrentExponent}}"] to
+        0.
+        </li>
+        <li>Set
+        |initialGlobalState|["{{HIDReportItem/unitFactorLuminousIntensityExponent}}"]
+        to 0.
+        </li>
+        <li>Set |initialGlobalState|["{{HIDReportItem/logicalMinimum}}"] to 0.
+        </li>
+        <li>Set |initialGlobalState|["{{HIDReportItem/logicalMaximum}}"] to 0.
+        </li>
+        <li>Set |initialGlobalState|["{{HIDReportItem/physicalMinimum}}"] to 0.
+        </li>
+        <li>Set |initialGlobalState|["{{HIDReportItem/physicalMaximum}}"] to 0.
+        </li>
+        <li>[=stack/Push=] |initialGlobalState| onto |globalStack|.
+        </li>
+        <li>[=list/For each=] |item| of |items|:
+          <div class="note">
+            <p>
+              [=Main items=] are described in [[USBIF-HID-CLASS]] sections
+              6.2.2.4 through 6.2.2.6. An item with the <dfn>Input tag</dfn>,
+              <dfn>Output tag</dfn>, or <dfn>Feature tag</dfn> defines a field
+              within an input, output, or feature report. The [=item data
+              field=] of a [=Main item=] with an [=Input tag=], [=Output tag=],
+              or [=Feature tag=] is a bit field containing several boolean
+              parameters describing the data field. An item with the
+              <dfn>Collection tag</dfn> defines a new collection nested within
+              the current collection, or a new [=top-level collection=] if
+              there is no current collection. The [=item data field=] of a
+              [=Main item=] with a [=Collection tag=] contains the
+              <dfn>collection type</dfn>. An item with the <dfn>End Collection
+              tag</dfn> closes the current collection.
+            </p>
+          </div>
+          <ol>
```

Comment:
This should probably be an unordered list, since it only has two entries for whether _item_ is a Main item or a Local item.